internal/graph/dotgraph.go

```diff
@@ -479,7 +480,10 @@ func min64(a, b int64) int64 {
 func escapeForDot(in []string) []string {
 	var out = make([]string, len(in))
 	for i := range in {
-		out[i] = strings.ReplaceAll(strings.ReplaceAll(strings.ReplaceAll(in[i], `\`, `\\`), `"`, `\"`), "\n", `\l`)
+		out[i] = escapeStringForDot(in[i])
 	}
 	return out
 }
+func escapeStringForDot(str string) string {
```

Comment:
I would call this function `escapeForDot` and rename the one above to `escapeAllForDot`.

Also, the comment from the function above should move to here since this is where it applies. And for the function above add a comment along the lines "applies escapeForDot to all strings in the given slice".

Reply:
Thanks @aalexand that's a good suggestion. I'll do that tomorrow! :)

Reply:
Thanks - please ping the PR once that is done.

Reply:
Thanks @aalexand - this is done! :)